### PR TITLE
Use rsync to transfer workdir to/from controller

### PR DESCRIPTION
**Pull request by bertsky**

### CODE REVIEW

File: ocr.sh [hunk 1]
```diff
@@ -95,11 +109,23 @@ pre_process_to_workdir () {
...
+
 ocrd_validate_workflow () {
     echo -n "ocrd validate tasks --workspace . "
     ocrd_format_workflow
 }
 
+post_sync_workdir () {
+    # copy the results back from Controller to Manager
+    rsync -avr --port $CONTROLLERPORT ocrd@$CONTROLLERHOST:/data/"$WORKDIR/" "$WORKDIR"
+    # TODO: maybe also schedule cleanup (or have a cron job delete dirs in /data which are older than N days)
```
Comment (markusweigelt) [inline ocr.sh:125]: Created a issue for that so we can remove todo #23

File: for_production.sh [hunk 2]
```diff
@@ -26,21 +26,17 @@ source ocr.sh
...
 
     ocrd_exec ocrd_import_workdir ocrd_validate_workflow ocrd_process_workflow
 
-    # TODO: copy the results back here
-    # e.g. `rsync -avr --port $CONTROLLERPORT ocrd@$CONTROLLERHOST:/data/"$WORKDIR" "$WORKDIR"`
-    # maybe also schedule cleanup (or have a cron job delete dirs in /data which are older than N days)
-    # e.g. `ssh --port $CONTROLLERPORT ocrd@$CONTROLLERHOST rm -fr /data/"$WORKDIR"`
+    post_sync_workdir
 
     post_process_validate_workdir
```
Comment (markusweigelt) [inline for_production.sh:41]: Should we rename  `post_process_validate_workdir` to `post_validate_workdir` for more continuity? Maybe   `pre_process_to_workdir ` but there i think it's necessary to know what happends.

File: ocr.sh (hunk 1 above)
Comment (bertsky) [reply ocr.sh:125]: Let's keep the comment though (since this is exactly where the fix should be made).

File: for_production.sh (hunk 2 above)
Comment (bertsky) [reply for_production.sh:41]: Agreed!

**Comment by bertsky** [comment]
Sorry, just saw that I started the PR branch too early. Will rebase against current master and force-push...

File: for_production.sh (hunk 2 above)
Comment (bertsky) [reply for_production.sh]: Done

**Comment by markusweigelt** [comment]
@bertsky can you merge the current master changes into this branch to resolve conflicts to finish pr?

**Comment by markusweigelt** [comment]
Sorry i have not seen the pr or forgot to merge it. We can rename the file again and then rename back after the merge again.

**Comment by markusweigelt** [comment]
I renamed the file in main but can you merge or trigger pr again for auto merge. In current state nothing seams to be changed in main repository for this pr.

**Comment by bertsky** [comment]
> I renamed the file in main but can you merge or trigger pr again for auto merge. In current state of this pr, nothing seams to be changed in main repository.

I was already in the middle of it. Aborting and reentering, I don't have to trigger the diff by hand, but the differ now does not even show sub-region differences anymore.

**Comment by bertsky** [comment]
IOW, please rename back. I'll redo my manual merge.

**Comment by bertsky** [comment]
Done!

**Pull request merged**